src/main/java/com/spartronics4915/frc2020/Constants.java

```diff
@@ -80,11 +85,11 @@
 
     public static final class PanelRotator
     {
-        public static final int kBeamSensorUpID = -1;
-        public static final int kBeamSensorDownID = -1;
+        public static final int kBeamSensorUpID = 5;
+        public static final int kBeamSensorDownID = 6;
 
-        public static final int kExtendMotorID = -1;
-        public static final int kSpinMotorID = -1;
+        public static final int kExtendMotorID = 7;
```

Comment:
These are incorrect - the Beam Sensor ids aren't determined yet, and the ExtendMotor and SpinMotor are 13 and 14, respectively.